.github/workflows/tfrobot_test.yml

```diff
@@ -0,0 +1,40 @@
+name: Tfrobot
+
+defaults:
+  run:
+    working-directory: tfrobot
+on:
+  schedule:
+    - cron: 0 3 * * *
```

Comment:
I think we should make it 4 instead to avoid failures from integration tests worflow